src/main/resources/application.yaml

```diff
@@ -0,0 +1,24 @@
+zeebe:
+  client:
+    max-execution-threads: 1000
+    evenly-allocated-max-jobs: 1000
+  #    max-execution-threads: 100
+  #    number-of-workers: 5
+  #    evenly-allocated-max-jobs: "#{${zeebe.client.max-execution-threads} / ${zeebe.client.number-of-workers}}"
+  broker:
+    contactpoint: "localhost:26500"
+
+timer: "PT45S"
+
+transaction-id-length: -1
+
+jws:
+  tenantKeys:
+    - name: "gorilla"
+      publicKey: "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAtPuP1m/jpt5O71c+5oNjfaZCCFuiz63936hoL2SfGvJtbRZjKZf5XHF8terfBzXb0W8HQ6o/204HOg/mm/fVGjdM29kiIB26u12hWNlJybqNvcUWDww0i54OkXPtvVoDG2DyTz6sCy5Y3sTxyVB5+hWs+lLeSyJ8cHzBH67HLfgsuZ+e3QNJkKtps/3Hkn7GE6gG4VApepdLNALvJY8v2l70Xkc4R2RRPhfBl5pWF+pFAUcXHRarybr+irjFLpM86ph/K1g/EuZ/LPiYGcizSR/U5jZhYDABzacPwEfqKILiLuzjCip3DpahfN0vZ/5Nel8+3y5zRVDDlamTqjjbiwIDAQAB"
```

Comment:
can you add a how-to guide for creating the keys?

Reply:
Where should I add the guide?